### PR TITLE
Added sad face for users who do not have images, included fixtures and instructions for loading fixtures, added logic to display emoticons only if user does not have an image

### PR DESCRIPTION
**Pull request by sean-lewis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Setup
 * Install Vagrant and Virtualbox
 * Initialize the vagrant box - `vagrant up`
 * Log into the server - `vagrant ssh`
+* Load fixture data `./manage.py loaddata "core/fixtures/core-test.json"` (optional)
 * Create a super user using the command `python manage.py createsuperuser`
```
Comment (kave) [inline README.md]: what does this do?

File: README.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ Setup
 * Install Vagrant and Virtualbox
 * Initialize the vagrant box - `vagrant up`
 * Log into the server - `vagrant ssh`
+* Load fixture data `./manage.py loaddata "core/fixtures/core-test.json"` (optional)
```
Comment (kave) [inline README.md:18]: what does this do?

Comment (sean-lewis) [inline README.md:18]: Fixture for easy import of data

Comment (m3brown) [inline README.md:18]: You might want to spell out this this creates sample data, as "optional" isn't very verbose.

**Pull request merged**